### PR TITLE
templates: add the "exo vm template delete" subcommand

### PR DESCRIPTION
**Pull request by mcorbin**
```
./exo -A prod vm template delete 52d6c0b0-a2a6-4513-a962-fb4f354fd7d8 26cc59ad-3cb7-4fe8-a2b6-946ec5bae97d
[+] Are you sure you want to delete template: "52d6c0b0-a2a6-4513-a962-fb4f354fd7d8" [yN]: y
[+] Are you sure you want to delete template: "26cc59ad-3cb7-4fe8-a2b6-946ec5bae97d" [yN]: y
delete template "52d6c0b0-a2a6-4513-a962-fb4f354fd7d8" ⠼                                 done!
delete template "26cc59ad-3cb7-4fe8-a2b6-946ec5bae97d" ⠼                                done!
```

### CODE REVIEW

File: cmd/template_delete.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			}
+			cmd := egoscale.DeleteTemplate{
+				ID: id,
+			}
+			if err != nil {
+				return err
+			}
+
+			if !force {
+				if !askQuestion(fmt.Sprintf("Are you sure you want to delete template: %q", arg)) {
```
Comment (falzm) [inline cmd/template_delete.go]: If you actually ask the user a question the message format should be:

```suggestion
				if !askQuestion(fmt.Sprintf("Are you sure you want to delete template %q?", arg)) {
```

File: cmd/template_delete.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+					continue
+				}
+			}
+
+			tasks = append(tasks, task{
+				cmd,
+				fmt.Sprintf("delete template %q", cmd.ID.String()),
```
Comment (falzm) [inline cmd/template_delete.go]: Looks like this is an async job, so the message should be:

```suggestion
				fmt.Sprintf("deleting template %q", cmd.ID.String()),
```

File: go.mod [hunk 3]
```diff
@@ -10,7 +10,7 @@ require (
 	github.com/cenkalti/backoff v2.0.0+incompatible
 	github.com/cpuguy83/go-md2man v1.0.8 // indirect
 	github.com/dustin/go-humanize v0.0.0-20180713052910-9f541cc9db5d
-	github.com/exoscale/egoscale v0.16.0
+	github.com/exoscale/egoscale v0.16.1-0.20190415122555-2143c08dc8f1
```
Comment (falzm) [inline go.mod]: Could you please release a tagged `v0.16.1` version? 😇

File: cmd/template_delete.go (hunk 1 above)
Comment (mcorbin) [reply cmd/template_delete.go]: Updated, thanks.

File: go.mod (hunk 3 above)
Comment (mcorbin) [reply go.mod]: Yep, I will do a release once https://github.com/exoscale/egoscale/pull/400 is merged (to also have this call in the release).

Comment (mcorbin) [reply go.mod]: done ;)

**Pull request merged**